src/Common/Locale.php

```diff
@@ -54,7 +64,7 @@ public function serialize(): string
     /**
      * @param string $locale
      *
-     * @return Locale
+     * @return static
      */
     public function unserialize($locale)
```

Comment:
This method seems to be missing some typehinting

Reply:
Declaration must be compatible with Serializable->unserialize(serialized: string)